extensions/amp-consent/0.1/amp-consent.css

```diff
@@ -146,6 +162,17 @@ amp-consent.i-amphtml-consent-ui-in.i-amphtml-consent-ui-iframe-fullscreen > .i-
   100% {opacity: 0.2;}
 }
 
+@keyframes lightbox-appear {
```

Comment:
Nit: `lightbox-appear` is a likely name to be overridden by author. Prefix `i-amphtml`.

Reply:
Changed to `i-amphtml-modal-appear`